### ISSUE TITLE
Read-only courses

### ISSUE THREAD

**Issue by taylortom** [adaptlearning/adapt_authoring#1859]
## Requested Feature
Ability to set courses as 'read-only', allowing users to view course content & preview, without being able to modify anything.

## Current Workaround
N/A

**Comment by canstudios-louisem** [comment]
How would this interact with shared? Would the owner also have to set shared? or would it be a 3rd option? 

Would the owner be able to edit it while it was read-only?

**Comment by taylortom** [comment]
Good point. I think we could probably do with redefining shared at the same time, it's always felt a bit half-baked.

**Comment by lc-thomasberger** [comment]
I think we need a (basic) review feature in core. 
I thought about a "review stage" where you stage a build course for review and allow others to comment. Staging a course would simply be copying a built course into a different location. So rather then downloading it, you move it somewhere on the server and serve the content over http. A staged course should be immutable to make commenting easier. This is easy to accomplish when you use a built course.
